l10n_it_ateco/__manifest__.py

```diff
@@ -1,47 +1,25 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
+# -*- coding: utf-8 -*-
 #
-#    Copyright (C) 2014 Abstract
-#    (<http://abstract.it>).
+#    Copyright (C) 2014 Abstract (<http://abstract.it>).
+#    Copyright (C) 2016 Ciro Urselli (<http://www.apuliasoftware.it>).
 #
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+#    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 {
     "name": "Ateco codes",
-    "version": "8.0.1.0.0",
-    "category": "Localisation/Italy",
-    "description": """Italian Localisation module - Ateco codes
-
-    Funcionalities:
-
-    - Add Ateco codes model
-    - Reference Ateco codes to partner model
-
-    """,
-    "author": "Abstract,Odoo Community Association (OCA)",
+    "version": "10.0.1.0.0",
+    "category": "Localization/Italy",
+    "author": "Abstract,Odoo Community Association (OCA),Odoo Italia Network",
     "website": "http://abstract.it",
     "license": "AGPL-3",
     "depends": [
-        "base"
+        "sales_team"
     ],
     "data": [
         "security/ir.model.access.csv",
         "view/ateco_view.xml",
         "view/partner_view.xml",
         "data/ateco_data.xml"
     ],
-    "active": False,
-    'installable': False
+    'installable': True
```

Comment:
Mixing single quotation and double quotation in the manifest. This would be nicer as:
`"installable": True`